contrib/go-chi/chi/chitrace_test.go

```diff
@@ -0,0 +1,142 @@
+package chi
+
+import (
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	dd "github.com/DataDog/dd-trace-go/opentracing"
+	"github.com/go-chi/chi"
+	opentracing "github.com/opentracing/opentracing-go"
+	"github.com/opentracing/opentracing-go/ext"
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTracingMiddleWare(t *testing.T) {
+	assert := assert.New(t)
+
+	// Tracing setup
+	tt := newTestTracer()
+	opentracing.SetGlobalTracer(tt)
+
+	// Router setup
+	r := chi.NewRouter()
+	r.Use(Middleware)
+
+	r.Get("/test/{id}", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(200)
+		fmt.Fprintln(w, "ᕕ( ᐛ )ᕗ")
+	})
+
+	req := httptest.NewRequest("GET", "/test/24601", nil)
+	res := httptest.NewRecorder()
```

Comment:
I think we should test the values of `res.Response()` here too. That the correct status and body is returned.